### DISCUSSION TITLE
Pacstall + AmogOS

### DISCUSSION THREAD

**Discussion by Elsie19** [pacstall/pacstall#335] [Show and tell]
https://github.com/Amog-OS/AmogOS/releases/tag/x64-1.4.0

We’ve done it gentleman. AmogOS has pacstall preinstalled.

Stay sussy
![](https://c.tenor.com/p-Lg-1C0eucAAAAC/sus-fry.gif)

**Comment by bonkisntbonk** [comment]
dumb

**Reply by Elsie19** [reply]
no
